WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -58,8 +58,13 @@ public async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo in
 
 		if (hasChange && !isFixedAmount && !info.IsPayJoin)
 		{
-			var suggestions =
-				ChangeAvoidanceSuggestionViewModel.GenerateSuggestionsAsync(info, destination, wallet, linkedCts.Token);
+			int originalInputCount = transaction.SpentCoins.Count();
+
+			// Use at most 5 coins.
+			int maxInputCount = (int)(Math.Max(3, originalInputCount * 1.3));
```

Comment:
Wouldn't `ChangelessTransactionCoinSelector.cs` be a better place for this line?